lib/generators/veteran.rb

```diff
@@ -89,7 +89,10 @@ def default_attrs(seed = nil)
                     entered_on_duty_date: "07022006",
                     released_active_duty_date: "06282008",
                     char_of_svc_code: "UHC",
-                    pay_grade: "E5" }]
+                    pay_grade: "E5" }],
+        state_of_residence: nil,
+        country_of_residence: nil,
+        residence_location_last_checked_at: nil
```

Comment:
```suggestion
                    pay_grade: "E5" }]
```
I am wondering, since this information isn't returned by BGS, if these three lines should be included here.

```ruby
Generators::Veteran.build(fl_vet_attrs).update(
  state_of_residence: "FL"
  country_of_residence: "USA",
  residence_location_last_checked_at: nil
).save!
```
I think something like this would allow the generator to operate truer to form.
```